Neos.Utility.Lock/Classes/TYPO3/Flow/Utility/Lock/Lock.php

```diff
@@ -13,17 +13,10 @@
 
 /**
  * A general lock class.
- *
- * @api
```

Comment:
Just noticed. Maybe this should stay now that the `acquire` method is in this class